### ISSUE TITLE
Multiset.size=NaN when removing non-existent element

### ISSUE THREAD

**Issue by kberg** [Yomguithereal/mnemonist#197]
```
    const multiset = new MultiSet();
    multiset.add('a');        console.log(multiset.size);
    multiset.add('a');        console.log(multiset.size);
    multiset.remove('a');     console.log(multiset.size);
    multiset.remove('a');     console.log(multiset.size);
    multiset.remove('a');     console.log(multiset.size);
```

Yields

```
1
2
1
0
NaN
```

If this is expected, consider documenting it accordingly.

**Comment by Yomguithereal** [comment]
Thanks @kberg. This is now fixed in v0.39.3.

**Comment by kberg** [comment]
Thank you!
